### ISSUE TITLE
Mac Ventura

### ISSUE THREAD

**Issue by NF-max** [inkstitch/inkstitch#3015]
I have had to re-install all my applications due to a new hard-drive, but now Ink Stitch will not work, when I go to Inkscape>extensions>params or visualiser/ simulator I get the spinning wheel of death and nothing happens, and then I get this warning
Exception in thread Thread-1:
Traceback (most recent call last):
  File "threading.py", line 932, in _bootstrap_inner
  File "lib/gui/simulator.py", line 790, in run
  File "lib/gui/simulator.py", line 811, in update_patches
  File "lib/stitch_plan/stitch_plan.py", line 35, in stitch_groups_to_stitch_plan
  File "lib/stitch_plan/stitch_plan.py", line 115, in new_color_block
  File "lib/stitch_plan/color_block.py", line 16, in __init__
  File "lib/stitch_plan/color_block.py", line 51, in color
  File "lib/threads/color.py", line 36, in __init__
TypeError: 'NoneType' object is not iterable

I've literally drawn one test line to see if it will work (bear in mind I have used the software before)

Please help, is it not compatible with VenturaOS?

Thank you

**Comment by NF-max** [comment]
No I haven’t, is that what I need to do?

**Comment by Marif91** [comment]
Quel est ton processseur ? Relis la doc que je t'ai mise...

**Comment by NF-max** [comment]
Im using Ventura, sorry if I am not responding correctly, my French is not great! Sent from my iPhoneOn 25 Jun 2024, at 5:28 PM, Marif91 ***@***.***> wrote:﻿
Quel est ton processseur ? Relis la doc que je t'ai mise...

**Comment by Marif91** [comment]
Si ton processeur est un M1 il faut installer Rosetta !

**Comment by NF-max** [comment]
It is not M1, and cannot install Rosetta
So what now?

**Comment by kaalleen** [comment]
macOS sometimes has this issue the colors. I think it is fixed for the next release.
So you can either try to re-apply your colors or try a dev build. We currently tested notarizarion for macOS, so this might be a good build to test (and easy to install): https://github.com/inkstitch/inkstitch/releases/tag/dev-build-kall-test-notarization
Pick the osx.pkg file and install as you normally would. We are shortly before a release. So if you encounter any errors with this release, it'd be nice to hear about them :)

**Comment by NF-max** [comment]
Ok, this is not helping
In my previous email I said I do not have M1 
So I cannot install Rosetta!
So what do I do now?

**Comment by tyrosinase** [comment]
Did you install the dev build kaalleen suggested?

**Comment by NF-max** [comment]
What did Kaalleen suggest? All I saw was the question of do I have Rosetta
installed, in which I replied my computer is saying it won’t allow it to be
installed.

I can’t preview anything in inkstitch, i can’t check params or check the
simulation. It won’t  load, i get the params window with all the settings
but i can’t check what the outcome will be, i just get no response from the
program and the spinning wheel of death. This is even with one single black
line, no design, just a test and even that crashes it, so I have no chance
with an actual design.
